src/user-guide/installing-workflows.rst

```diff
@@ -67,8 +67,8 @@ addition of the option ``--directory=PATH/TO/SOURCE/DIRECTORY`` (alternatively,
 .. note::
 
    To avoid confusion, ``cylc install`` will fail if the workflow name
-   contains ``runN`` or ``run<number>``. These directory names are
-   reserved for the automatic numbering of run directories.
+   contains directory names that are reserved filenames, such as
+   ``log``, ``work``, ``runN``, ``run<number>``, ``_cylc-install`` etc.
```

Comment:
Wonder if we should put a full list of forbidden names rather than an etc.. Just for user clarity?

Reply:
Best I can do is
```rst
   .. autoattribute:: cylc.flow.workflow_files.WorkflowFiles.RESERVED_NAMES
```
But it isn't especially neat, nor does it include `run<number>`

![image](https://user-images.githubusercontent.com/61982285/145996862-5178d9df-747d-430c-843f-9618ec1e6b45.png)

Reply:
Yes, I agree it isn't especially neat or elegant with the run number not included :(
Although personally think it best to be comprehensive in documentation, so if it were up to me I'd take the auto linked code. I'll leave it up to you @MetRonnie/second reviewer to decide.

Reply:
Done